frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -97,6 +97,11 @@ void main(List<String> args) async {
       expect(listOfStructResp[3].width, 420);
     }
 
+    print('dart call handleStringList');
+    {
+      await api.handleStringList(names: ['Steve', 'Bob', 'Alex']);
```

Comment:
What about making a very simple assert to its return value, e.g. `expect(..., ...)`